diff --git a/node_modules/@oclif/core/lib/config/config.js b/node_modules/@oclif/core/lib/config/config.js
index fca8f1a..606bf2c 100644
--- a/node_modules/@oclif/core/lib/config/config.js
+++ b/node_modules/@oclif/core/lib/config/config.js
@@ -150,6 +150,7 @@ class Config {
         }
     }
     async loadDevPlugins() {
+      if (this.options.devPlugins !== false) {
         // do not load oclif.devPlugins in production
         if (this.isProd)
             return;
@@ -161,6 +162,7 @@ class Config {
         catch (error) {
             process.emitWarning(error);
         }
+      }
     }
     async loadUserPlugins() {
         if (this.options.userPlugins !== false) {
